routes/_components/dialog/components/ImageDialog.html

```diff
@@ -30,21 +36,26 @@
     max-height: calc(100% - 20px);
     overflow: hidden;
   }
+
+  .autoplay-video {
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+  }
```

Comment:
We probably shouldn't repeat CSS selectors across multiple files, as I actually flatten all CSS for the production build (i.e. no per-component encapsulation) because it's smaller that way.